gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaAvroExtractor.java

```diff
@@ -42,8 +46,11 @@
 public class KafkaAvroExtractor extends KafkaExtractor<Schema, GenericRecord> {
 
   private static final Logger LOG = LoggerFactory.getLogger(KafkaAvroExtractor.class);
+  private static final Schema DEFAULT_SCHEMA = SchemaBuilder.record("PageViewEvent").fields().name("header")
```

Comment:
The schema needs to have a `header.time` field, since this is needed to create `AvroHdfsTimePartitionedWriter`.

I can change the name to "DefaultSchema"